infra/embedding_step_functions/simple_lambdas/find_unembedded/handler.py

```
@@ -4,21 +4,110 @@
+    """
+    Serialize ReceiptLines into per-receipt NDJSON files.
+    """
+    results: list[dict] = []
+    for image_id, receipts in line_receipt_dict.items():
+        for receipt_id, lines in receipts.items():
+            # Serialize each line as JSON
+            ndjson_lines = [json.dumps(dict(line)) for line in lines]
+            ndjson_content = "\n".join(ndjson_lines)
+            # Write to a unique NDJSON file
+            filepath = Path(
+                f"/tmp/{image_id}_{receipt_id}_lines_{uuid4()}.ndjson"
```

Ruff (0.14.6)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)